scene/main/viewport.cpp

```diff
@@ -3315,6 +3315,9 @@ void Viewport::_subwindow_visibility_changed() {
 	gui.subwindow_visibility_dirty = true;
 }
 
+float Viewport::editor_tooltip_delay = 0.5f;
+float Viewport::regular_tooltip_delay = 0.5f;
```

Comment:
Since the `master` branch now accepts C++17 code, you can initialize those variables directly in the header file.

Also, is `static` needed here? We only make variables `static` if we absolutely need it.